src/views/catalog/customize/components/CustomizeSource/ExpandableCustomizeSourceSection.scss

```diff
@@ -0,0 +1,11 @@
+.expandable-customize-source-section__stack-item-storage {
+  .pf-l-flex {
+    --pf-l-flex--AlignItems: center;
+  }
+}
+
+.expandable-customize-source-section__expandable-section-storage {
+  .pf-c-expandable-section__toggle {
+    padding-right: var(--pf-c-expandable-section__toggle--PaddingTop);
+  }
+}
```

Comment:
this looks good and definitely will work
another suggestion to improve, so we could remove the className from the ExpandableSectionToggle component:
```
.expandable-customize-source-section {
  .pf-l-flex {
    --pf-l-flex--AlignItems: center;
  }

  .pf-c-expandable-section__toggle {
    padding-right: var(--pf-c-expandable-section__toggle--PaddingTop);
  }
}
```